scraper.py

```diff
@@ -156,6 +193,7 @@ def get_elections():
                             'url': "https://candidates.democracyclub.org.uk/election/%s/constituencies" % (election_id),
                             'post_id': None,
                             'locked': False,
+                            'sopn_published': str(sopn_date)
```

Comment:
Casting to string here (and a few lines down) is going to give us odd behaviour given `get_sopn_date()` can return `None` and python will just merrily compare strings to each other with `<` or `>`:

```py
>>> str(None)
'None'

>>> 'None' > '2019-08-14'
True
```

also see above comment about representing this as a `DATETIME` - think that's going to be less error prone

Reply:
This should be less of a problem with the explicit None-checks against `sopn_published` now.